lib/Conversion/TorchToTosa/TorchToTosa.cpp

```diff
@@ -3027,6 +3027,37 @@ LogicalResult ConvertAtenOp<AtenWhereSelfOp>::matchAndRewrite(
   return success();
 }
 
+template <>
+LogicalResult ConvertAtenOp<AtenClampOp>::matchAndRewrite(
+    AtenClampOp op, OpAdaptor adaptor,
+    ConversionPatternRewriter &rewriter) const {
+
+  // Not a tensor type.
+  auto selfType = adaptor.self().getType().dyn_cast<TensorType>();
+  if (!selfType)
+    return rewriter.notifyMatchFailure(
+        op, "Only tensor types input are currently supported");
```

Comment:
nit: A very tiny question. Forget to start the first sentence with a lower-case letter?

Reply:
good catch, yes, to be consistent, "Only" -> "only"